sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConstants.cs

```diff
@@ -24,14 +24,14 @@ internal class AmqpMessageConstants
         internal const string UriName = AmqpConstants.Vendor + ":uri";
         internal const string DateTimeOffsetName = AmqpConstants.Vendor + ":datetime-offset";
         /// <summary>
-        ///  Property key representing deadletter reason, when a message is received from a deadletter subqueue of an entity.
+        ///  Property key representing dead-letter reason, when a message is received from a dead-letter sub-queue of an entity.
         ///  This key and the associated values are stored in the <see cref="ServiceBusReceivedMessage.ApplicationProperties"/> dictionary
         ///  for dead lettered messages.
         /// </summary>
         internal const string DeadLetterReasonHeader = "DeadLetterReason";
 
         /// <summary>
-        ///  Property key representing detailed error description, when a message is received from a deadletter subqueue of an entity.
+        ///  Property key representing detailed error description, when a message is received from a dead-letter sub-queue of an entity.
```

Comment:
subqueue matches the service documentation - https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dead-letter-queues

Reply:
```suggestion
        ///  Property key representing detailed error description, when a message is received from a dead-letter subqueue of an entity.
```